### ISSUE TITLE
Errors swallowed in tests

### ISSUE THREAD

**Issue by kiwiupover** [jherdman/ember-cli-trackjs#54]
We are using trackjs along with ember-cli-mocha and are having random test failures.

After spending most of a day looking into the issue then having this Ray's blog post [ember-onerror-troll](http://raytiley.com/posts/ember-onerror-troll) arrive in this weeks [emberweekly 146](http://emberweekly.com/) I now see where the errors are being swallowed. 

This [line](https://github.com/jherdman/ember-cli-trackjs/blob/master/app/instance-initializers/configure-trackjs.js#L16) is problematic in tests. 
```js
  Ember.onerror = handler.report.bind(handler);
``` 

Here are two solutions:
In `instance-initializers/configure-trackjs.js`
Either:
```js
 if (Ember.testing) { return; } // we don't need to be sending errors to trackjs
```

```js
if (Ember.testing !== true) {
   Ember.onerror = handler.report.bind(handler);
}
```

Would you like a PR?

### 1.
```js
import Ember from 'ember';
import ErrorHandler from '../utils/error-handler';

export function initialize(app) {
  if (Ember.testing) { return; } // we don't need to be sending errors to trackjs

  const instance = app.lookup ? app : app.container;

  const trackJs = instance.lookup('service:trackjs');
  const appVersion = instance.lookup('application:main').get('version');

  trackJs.configure({
    version: appVersion
  });

  const handler = new ErrorHandler(trackJs);

  Ember.onerror = handler.report.bind(handler);
}

export default {
  name: 'trackjs-error-and-serializer-configuration',
  initialize: initialize
}
```

### 2.

```js
import Ember from 'ember';
import ErrorHandler from '../utils/error-handler';

export function initialize(app) {
  if (Ember.testing) { return; }

  const instance = app.lookup ? app : app.container;

  const trackJs = instance.lookup('service:trackjs');
  const appVersion = instance.lookup('application:main').get('version');

  trackJs.configure({
    version: appVersion
  });

  if (Ember.testing !== true) { 
    const handler = new ErrorHandler(trackJs);
    Ember.onerror = handler.report.bind(handler);
  }
}

export default {
  name: 'trackjs-error-and-serializer-configuration',
  initialize: initialize
}
```

**Comment by jherdman** [comment]
Well, shit. Look at that! I'm sorry to hear you were being trolled so hard by this. I'd love a PR!

**Comment by jherdman** [comment]
Hi @kiwiupover ! I thought I'd check in and see if you're still interested in submitting a PR for your proposed changes.

**Comment by kiwiupover** [comment]
I will get that done today.

**Comment by jherdman** [comment]
Closed via #55
